lesson_11/arrays_java/arrays_app/src/main/java/com/codedifferently/lesson11/Lesson11.java

```diff
@@ -9,14 +10,32 @@ public class Lesson11 {
    * https://leetcode.com/problems/concatenation-of-array
    */
   public int[] getConcatenation(int[] nums) {
-    return null;
-  }
+    int n = nums.length;
+    int[] ans = new int[n*2];
+
+    for (int i=0; i<n; i++) {
+      ans[i]=nums[i];
+      ans[i+n]=nums[i];
+      } return ans;
```

Comment:
Need to run `./gradlew spotlessApply` to fix the formatting issues in this file.